auth/jwt/middleware.go

```diff
@@ -0,0 +1,106 @@
+package jwt
+
+import (
+	"errors"
+
+	"golang.org/x/net/context"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/go-kit/kit/endpoint"
+)
+
+type contextKey string
+
+const (
+	// JWTTokenContextKey holds the key used to store a JWT Token in the context
```

Comment:
Please add a period to the end of the comment sentence, and wrap at 80 col if it isn't already.